documentation/Functions.md

```diff
@@ -240,11 +241,27 @@ image: Path to an image to be displayed in the katacoda step.
 
 ***
 
-
 ### adaptTemplatesCobiGen
 #### parameter
 * No parameters
 #### example
 adaptTemplatesCobiGen()
 
 ***
+
+### createDevon4ngProject
+#### parameter 
+1. Name of the Project.
+2. Path to where the Project should be created (relative to workspace). Folder should exist.
+3. (Optional) Optional parameters, according to https://angular.io/cli/new .
```

Comment:
We have to mention in what format the user have to pass the parameters. In you current implemention it has to be a string. I think a string array like in npmInstall would be better.